Password Verify/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Problem
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            String pass;
```

Comment:
Read about [SecureString class](https://docs.microsoft.com/en-us/dotnet/api/system.security.securestring?view=net-6.0), and use it.